code/src/hardware_usart.cpp

```diff
@@ -3,128 +3,136 @@
 
 namespace r2d2 {
 
-    bool hardware_usart_c::transmit_ready() {
-        return (hardware_usart->US_CSR & 2);
-    }
+bool hardware_usart_c::transmit_ready() { return (hardware_usart->US_CSR & 2); }
```

Comment:
Single line if-statements are not accepted in the style guide. While it is silent on single line functions it would be good practice to stay on the same track